### DISCUSSION TITLE
縦列のスイッチが反応しない

### DISCUSSION THREAD

**Discussion by ruby620** [Yowkees/keyball#281] [Q&A]
![右側基板01](https://github.com/Yowkees/keyball/assets/140152065/f19109b7-1d8d-4e24-bba5-b73507b0ff39)

■右側基板のスイッチ反応しない件について
１．赤四角①と②の縦列のスイッチが反応しません。（それ以外は動作問題ありません）

２．青四角①は全て導通しており、COL0と導通してます。

３．青四角②は全て導通していますが、promicroどのピンとも導通取れていません。
　　（promicroの何番ピンにつながるのでしょうか？）

４．赤四角①と②のスイッチ動作させるための調査方法をご教授ください。

５．keyball61　rev2使用してます

６．keyball61　rev2の回路図を公開してもらえないでしょうか？

７．promicroのストラップ線は、１２ピンソケットを半田付け時に半田を盛りすぎてピンが接触する側まで
　　半田が上がってしまいピンヘッダが刺さらなくなったため該当箇所をストラップ線で貼りました。

![左側基板01](https://github.com/Yowkees/keyball/assets/140152065/e3f0bcbf-f7ae-4b8d-a9e9-e4daddf94878)

■左側基板のスイッチ反応しない件について
１．赤四角①と②と③の縦列のスイッチが反応しません。（それ以外は動作問題ありません）

２．青四角①は全て導通しており、COL２と導通してます。

３．青四角②は全て導通していますが、promicroどのピンとも導通取れていません。
　　（promicroの何番ピンにつながるのでしょうか？）

４．青四角③は全て導通していますが、promicroどのピンとも導通取れていません。
　　（promicroの何番ピンにつながるのでしょうか？）

５．赤四角①と②と③のスイッチ動作させるための調査方法をご教授ください。

６．keyball61　rev2使用してます

７．keyball61　rev2の回路図を公開してもらえないでしょうか？

８．promicroのストラップ線は、１２ピンソケットを半田付け時に半田を盛りすぎてピンが接触する側まで
　　半田が上がってしまいピンヘッダが刺さらなくなったため該当箇所をストラップ線で貼りました。

**Comment by ruby620** [comment]
とりあえず原因が分かり動くようになりました。原因は下記の通りでした。

1.右側基板の縦列スイッチが反応しない件は、COL0のピンソケットとプロマイクロのピンが接触してないのが原因でした。

2.左側基板の縦列スイッチが反応しない件は、COL2のピンソケットとプロマイクロのピンが接触してないのが原因でした。

3.私移行12ピンソケットに半田が上がりすぎてピンヘッダが刺さらなくなった人が居るみたいですが半田の盛りすぎ注意です。

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。
ご連絡が遅くなり、ご自身で解決していただいたようで、誠にありがとうございました。

回路図と基板データは下記にて公開しておりますので、今後ご活用いただければ幸いです。
https://github.com/Yowkees/keyball/tree/main/keyball61/design_data

以上、今後ともどうぞよろしくお願いいたします。
